IfcDoor Rail Import BIMQ 2021-07-21T06:45:53.5338603Z (#159)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TicketProcessing/Documentation.md

> Properties in this property set indicate performance ratings of entry elements (e.g. turnstile, boom barrier) on processing tickets.

Reviewer: ```suggestion
Properties for indicating performance ratings for ticket processing of entry elements (e.g. turnstile, boom barrier).
```
